pulp_rpm/app/migrations/0002_rpm_related_names.py

```diff
@@ -0,0 +1,106 @@
+# Generated by Django 2.2.4 on 2019-08-09 19:10
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0003_remove_upload_completed'),
+        ('rpm', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ModulemdDefaults',
+            fields=[
+                ('content_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, related_name='rpm_modulemddefaults', serialize=False, to='core.Content')),
+                ('module', models.CharField(max_length=255)),
+                ('stream', models.CharField(max_length=255)),
+                ('profiles', models.TextField(default='[]')),
+            ],
+            options={
+                'default_related_name': '%(app_label)s_%(model_name)s',
+            },
+            bases=('core.content',),
+        ),
+        migrations.AlterModelOptions(
+            name='package',
+            options={'default_related_name': '%(app_label)s_%(model_name)s'},
+        ),
+        migrations.AlterModelOptions(
+            name='rpmdistribution',
+            options={'default_related_name': '%(app_label)s_%(model_name)s'},
+        ),
+        migrations.AlterModelOptions(
+            name='rpmpublication',
+            options={'default_related_name': '%(app_label)s_%(model_name)s'},
+        ),
+        migrations.AlterModelOptions(
+            name='rpmremote',
+            options={'default_related_name': '%(app_label)s_%(model_name)s'},
+        ),
+        migrations.AlterModelOptions(
+            name='updaterecord',
+            options={'default_related_name': '%(app_label)s_%(model_name)s'},
```

Comment:
@daviddavis @fabricio-aguiar 

I would have thought that it should be formatting the string prior to the migration step, so that the migration would contain the true name of the field as represented inside the database.  Is that not actually the case?

Reply:
The `default_related_name` option doesn't actually correspond to a field in the database so I am not surprised. You can see though that creating the migration actually resolved `default_related_name` below in the `AlterField` operations. For example, [`package` becomes `rpm_package` on `Content`](https://github.com/pulp/pulp_rpm/pull/1417/files#diff-2554c1ca07ddb97699cb54a8620ca6abR61).